package.json

```diff
@@ -303,6 +303,7 @@
 		]
 	},
 	"workspaces": [
-		"packages/*"
+		"packages/*",
+		"routes/*"
```

Comment:
Interesting. How will this be consumed in core?

Reply:
This is a good point that I'm not sure about yet :P Any ideas? 

 - We could "ship" as packages potentially or a git submodule.